packages/infrastructure/authentik_atlas_mongodb_sso/vars.tf

```diff
@@ -0,0 +1,46 @@
+variable "media_configmap" {
+  description = "The configmap holding the static media that Authentik will use"
+  type        = string
+}
+
+variable "authentik_namespace" {
```

Comment:
should have a default

Reply:
not following why a default? I think it should be required that a namespace is provided no?

Reply:
To make setup easier, the default should just be `authentik` as that is the already the provided default for the `kube_authentik` module.